recipes-core/aziot-edged/aziot-edged.inc

```diff
@@ -40,6 +40,20 @@ do_install () {
 
 }
 
+pkg_postinst_ontarget_${PN}() {
+    #!/bin/sh
+
+    # Add each admin user to the iotedge group
+    for u in $(getent group admin | sed -e "s/^.*://" -e "s/,/ /g"); do
```

Comment:
@ggjjj / @maskimthedog will `getent` be available in the image to use on first boot or will that depend on other things someone chooes to include?

Reply:
@micahl @ggjjj getent is in glibc-utils.

Reply:
Should the RDEPENDS then include 'glibc-utils' in the list?  Otherwise, when I try the change as-is for my RPi3 there is no 'getent' command.

Reply:
> Should the RDEPENDS then include 'glibc-utils' in the list? Otherwise, when I try the change as-is for my RPi3 there is no 'getent' command.

Yes, it should.  It was in the RDEPENDS on my fork.  Looked like it did not make it into this PR.

Reply:
![image](https://user-images.githubusercontent.com/41034265/213607048-4a44b2f3-592e-45ee-8f16-912ea4e8fe8f.png)

Reply:
@ggjjj once glibc-utils is added then I think the change looks good